terraform/groups/document-signing-api/data.tf

```
@@ -23,7 +24,8 @@ data "aws_iam_policy_document" "image_bucket" {
     ]
 
     resources = [
-      "arn:aws:s3:::${var.image_bucket_name_prefix}-*/*"
+      "arn:aws:s3:::${var.image_bucket_name_prefix}-*/*",
+      "arn:aws:s3:::ch-${var.image_bucket_name_prefix}-*/*"
```

As above, revert these changes. Overriding the `image_bucket_name_prefix` variable default value should be enough:

```suggestion
      "arn:aws:s3:::${var.image_bucket_name_prefix}-*/*"
```

---

Made this change too.